Assets/HoloToolkit/Build/Editor/HoloToolkitCommands.cs

```diff
@@ -1,4 +1,9 @@
-﻿using HoloToolkit.Unity;
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Don't you work at Microsoft and wrote this in the course of work on HoloLens? You may want to review your employment terms that likely automatically claim ownership in this case. (You checked the second box in the CLA, right? https://cla.microsoft.com/cladoc/microsoft-contribution-license-agreement.pdf)